### ISSUE TITLE
Exception at startup

### ISSUE THREAD

**Issue by Stefku** [ziolko/intellij-javascript-symbols#1]
I got an exception at startup of intellij

I use IntelliJ IDEA 15.0.3
Build #IU-1431821
JRE: 1.7.0_67-b01 amd64

```
Plugin 'com.webstorm.symbols' failed to initialize and will be disabled.  Please restart IntelliJ IDEA.

com.intellij.diagnostic.PluginException: com/webstorm/symbols/SymbolPluginInitializer : Unsupported major.minor version 52.0 [Plugin: com.webstorm.symbols]
    at com.intellij.ide.plugins.cl.PluginClassLoader.a(PluginClassLoader.java:130)
    at com.intellij.ide.plugins.cl.PluginClassLoader.a(PluginClassLoader.java:77)
    at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:66)
    at java.lang.ClassLoader.loadClass(ClassLoader.java:358)
    at java.lang.Class.forName0(Native Method)
    at java.lang.Class.forName(Class.java:270)
    at com.intellij.openapi.components.impl.ComponentManagerImpl.a(ComponentManagerImpl.java:408)
    at com.intellij.openapi.components.impl.ComponentManagerImpl.init(ComponentManagerImpl.java:96)
    at com.intellij.openapi.application.impl.ApplicationImpl.load(ApplicationImpl.java:455)
    at com.intellij.openapi.application.impl.ApplicationImpl.load(ApplicationImpl.java:447)
    at com.intellij.idea.IdeaApplication.run(IdeaApplication.java:194)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:606)
    at com.intellij.idea.MainImpl$1$1$1.run(MainImpl.java:60)
    at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:312)
    at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:733)
    at java.awt.EventQueue.access$200(EventQueue.java:103)
    at java.awt.EventQueue$3.run(EventQueue.java:694)
    at java.awt.EventQueue$3.run(EventQueue.java:692)
    at java.security.AccessController.doPrivileged(Native Method)
    at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:76)
    at java.awt.EventQueue.dispatchEvent(EventQueue.java:703)
    at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:361)
    at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:242)
    at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:161)
    at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:150)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:146)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:138)
    at java.awt.EventDispatchThread.run(EventDispatchThread.java:91)
Caused by: java.lang.UnsupportedClassVersionError: com/webstorm/symbols/SymbolPluginInitializer : Unsupported major.minor version 52.0
    at java.lang.ClassLoader.defineClass1(Native Method)
    at java.lang.ClassLoader.defineClass(ClassLoader.java:800)
    at java.lang.ClassLoader.defineClass(ClassLoader.java:643)
    at com.intellij.util.lang.UrlClassLoader._defineClass(UrlClassLoader.java:260)
    at com.intellij.util.lang.UrlClassLoader.defineClass(UrlClassLoader.java:256)
    at com.intellij.util.lang.UrlClassLoader._findClass(UrlClassLoader.java:225)
    at com.intellij.ide.plugins.cl.PluginClassLoader.a(PluginClassLoader.java:124)
    ... 30 more
```

**Comment by ziolko** [comment]
Hi Stefku,

Thank you for the bug report. The project was compiled with Java language level 1.8.
I've deployed a new version targeting Java 6 so it should work now. Please let me know if it's fine.

**Comment by Stefku** [comment]
Hi Ziolko, thanks for the quick fix. Unfortunately, I get another Exception, but apparently, it's getting closer :)

```
Plugin 'com.webstorm.symbols' failed to initialize and will be disabled.  Please restart IntelliJ IDEA.

java.lang.NoClassDefFoundError: java/util/function/Consumer
    at com.webstorm.symbols.index.JSSymbolsIndex.getInputFilter(JSSymbolsIndex.java:72)
    at com.intellij.util.indexing.FileBasedIndexImpl.a(FileBasedIndexImpl.java:438)
    at com.intellij.util.indexing.FileBasedIndexImpl.a(FileBasedIndexImpl.java:410)
    at com.intellij.util.indexing.FileBasedIndexImpl.a(FileBasedIndexImpl.java:296)
    at com.intellij.util.indexing.FileBasedIndexImpl.initComponent(FileBasedIndexImpl.java:373)
    at com.intellij.openapi.components.impl.ComponentManagerImpl$ComponentConfigComponentAdapter.getComponentInstance(ComponentManagerImpl.java:518)
    at com.intellij.openapi.components.impl.ComponentManagerImpl.createComponents(ComponentManagerImpl.java:123)
    at com.intellij.openapi.application.impl.ApplicationImpl.access$801(ApplicationImpl.java:91)
    at com.intellij.openapi.application.impl.ApplicationImpl$12.run(ApplicationImpl.java:497)
    at com.intellij.openapi.progress.impl.CoreProgressManager$2.run(CoreProgressManager.java:142)
    at com.intellij.openapi.progress.impl.CoreProgressManager.a(CoreProgressManager.java:446)
    at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:392)
    at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:54)
    at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:127)
    at com.intellij.openapi.application.impl.ApplicationImpl.createComponents(ApplicationImpl.java:512)
    at com.intellij.openapi.components.impl.ComponentManagerImpl.init(ComponentManagerImpl.java:107)
    at com.intellij.openapi.application.impl.ApplicationImpl.load(ApplicationImpl.java:455)
    at com.intellij.openapi.application.impl.ApplicationImpl.load(ApplicationImpl.java:447)
    at com.intellij.idea.IdeaApplication.run(IdeaApplication.java:194)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:606)
    at com.intellij.idea.MainImpl$1$1$1.run(MainImpl.java:60)
    at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:312)
    at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:733)
    at java.awt.EventQueue.access$200(EventQueue.java:103)
    at java.awt.EventQueue$3.run(EventQueue.java:694)
    at java.awt.EventQueue$3.run(EventQueue.java:692)
    at java.security.AccessController.doPrivileged(Native Method)
    at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:76)
    at java.awt.EventQueue.dispatchEvent(EventQueue.java:703)
    at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:361)
    at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:242)
    at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:161)
    at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:150)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:146)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:138)
    at java.awt.EventDispatchThread.run(EventDispatchThread.java:91)
Caused by: java.lang.ClassNotFoundException: java.util.function.Consumer PluginClassLoader[com.webstorm.symbols, 1.3]
    at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:68)
    at java.lang.ClassLoader.loadClass(ClassLoader.java:358)
    ... 39 more
```

**Comment by ziolko** [comment]
It seems I was too optimistic when it comes to compatibility with Java 7. 
I've fixed the mentioned issue so please try again with the newest version.

Sorry for the inconvenience and feel free to report other bugs and request features. I can't promise anything but I will try to add/fix things from time to time :)

**Comment by Stefku** [comment]
Ziolko, I had no inconvenience. I'm very glad that people like you have good ideas and also implement them :-)

Anyway, my problem ist solved with version 1.4. Thanks alot.
